doc/whats_new/v1.1.rst

```diff
@@ -229,6 +229,14 @@ Changelog
   :pr:`21481` by :user:`Guillaume Lemaitre <glemaitre>` and
   :user:`Andrés Babino <ababino>`.
 
+- |Fix| :class:`linear_model.LogisticRegression` and
+  :class:`linear_model.LogisticRegressionCV` now set the `n_iter_` attribute
+  with a shape that respects the docstring and that is consistent with the shape
+  obtained when using the other solvers in the one-vs-rest setting. Previously,
+  it would record only the maximum of the number of iterations for each binary
+  sub-problem was returned while now all of them are recorded. :pr:`21998` by
```

Comment:
There's something weird in this sentence
```suggestion
  obtained when using the other solvers in the one-vs-rest setting. Previously,
  it would record only the maximum of the number of iterations for each binary
  sub-problem while now all of them are recorded. :pr:`21998` by
```